region/maridia/outer.json

```diff
@@ -3869,6 +3869,21 @@
                   "note": "Morph and unmorph then drift to the ledge.",
                   "devNote": "Strat starts before entering this transition. An enemy can be used for this strat instead of morph."
                 },
+                {
+                  "name": "Fish Tank Wall Jump Entry",
+                  "notable": true,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canPreciseWalljump",
+                    "canPrepareForNextRoom",
```

Comment:
I'm going with `canCarefulJump` now.

And I don't think you can physics check with `adjacentRunway` on a vertical door since it won't have a defined runway.
So give them `canMidairWiggle` with a fixme for later.

The physics check would be a `adjacentJumpway` thing, but those don't have that yet either.  So it'd mean adding `doorFrameAbove` and `physics` as fields and values to jumpways.